bundles/org.openhab.binding.nikohomecontrol/src/main/java/org/openhab/binding/nikohomecontrol/internal/handler/NikoHomeControlEnergyMeterHandler.java

```diff
@@ -104,6 +104,13 @@ public void initialize() {
 
             updateStatus(ThingStatus.ONLINE);
```

Comment:
I would assume this status update can be removed with the code added below.

Reply:
Correct, it is not.